src/coreComponents/schema/docs/InternalMesh.rst

```diff
@@ -10,7 +10,7 @@ nx                integer_array required Number of elements in the x-direction w
 ny                integer_array required Number of elements in the y-direction within each mesh block                                            
 nz                integer_array required Number of elements in the z-direction within each mesh block                                            
 positionTolerance real64        1e-10    A position tolerance to verify if a node belong to a nodeset                                            
-trianglePattern   integer       0        Pattern by which to decompose the hex mesh into prisms (more explanation required)                      
+trianglePattern   integer       0        Pattern by which to decompose the hex mesh into wedges (more explanation required)                      
```

Comment:
Probably not your job to solve this, but I wonder if we can add the additional explanation as requested.  May have to ask Chris, Randy, or Pengcheng for some insight.  Otherwise, maybe just delete the parenthetical expression.

Reply:
Addressed adding a sketch to the doxygen documentation

https://github.com/GEOSX/GEOSX/blob/4ec7d440f9e76cfa2022139676d846b88ec63794/src/coreComponents/mesh/generators/InternalMeshGenerator.hpp#L200-L237